### PR TITLE
[AND-2388] AdMob improvements

### PR DESCRIPTION
**Pull request by HuFangshuai**
This PR have been created for https://vungle.atlassian.net/browse/AND-2388

### CODE REVIEW

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java [hunk 1]
```diff
@@ -241,15 +240,7 @@ public void requestBannerAd(Context context,
             return;
         }
 
-        //workaround for missing onPause/onResume/onDestroy
-        adLayout = new RelativeLayout(context) {
-            @Override
-            protected void onWindowVisibilityChanged(int visibility) {
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java]: From AdMob SDK 18.2.0 all devices can receive onPause() and onResume() callback. So onWindowVisibilityChanged() doesn't needed.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java [hunk 2]
```diff
@@ -241,15 +240,7 @@ public void requestBannerAd(Context context,
             return;
         }
 
-        //workaround for missing onPause/onResume/onDestroy
```
Comment (alexander-gubov) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java]: @HuFangshuai why did you remove this? Did callbacks start to work on all android APIs?

Comment (HuFangshuai) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java]: @alexander-gubov 

> @HuFangshuai why did you remove this? Did callbacks start to work on all android APIs?
Thanks your comments. I tested on API 28 & 29  and callbacks works well. I am working on testing on more Android APIs.
